### ISSUE TITLE
Add option to az.plot_pair to show points from different chains as different colors

### ISSUE THREAD

**Issue by bwengals** [arviz-devs/arviz#2348]
It'd be nice to add an option to `plot_pair` that would show each chain as a different color.  This would help users diagnose whether chains have converged and whether different chains have settled into different modes.

**Comment by aloctavodia** [comment]
Indeed! This is something that should be easier to do with the new ArviZ (https://github.com/arviz-devs/arviz-plots), once we have `plot_pair`.

**Comment by imperorrp** [comment]
@bwengals @aloctavodia I've noted this! Would be good to have in the new `plot_pair` functionality
